### ISSUE TITLE
Create a shorthand for removing all listeners from event

### ISSUE THREAD

**Issue by arturmuller** [luskhq/redux-ws#4]
This should be possible:

```
unsubscribe('postCreated');
```

...should call `socket.removeAllListeners('postCreated')`.

**Comment by StevenLangbroek** [comment]
Maybe you can handle this the way most promise middleware does this? Just a thought :)

``` js
const unsubscribeFromPostCreation = () => ({
  type: 'DO_THING',
  meta: {
    manageSocket: (socket) => {
      socket.removeAllListeners('postCreated');
    }
  }
});
```

**Comment by arturmuller** [comment]
Hey Steven! How are you doing mate?

Good question. Honest answer: I didn't think of it. 😂 

I would definitely like to be 'compatible' with the Redux ecosystem. This was a simple idea I had when we  first started working with sockets and Redux, but couple of days ago I noticed that it gathered some stars, so I polished it enough to be considered a real package (with fancy things like README and stuff!).

I think for quick mockups/getting started with sockets it is actually not a bad solution because it is so simple (similarly like [Redux Thunk](https://github.com/gaearon/redux-thunk)), but for more complex apps you probably want to use [Redux Saga](https://github.com/yelouafi/redux-saga) in combination with some custom socket service... Or something like that.

I will definitely consider your proposed concept though.
